### DISCUSSION TITLE
Rustdesk Nightly Version is perceived by Windows Defender as Trojan. False virus alarm

### DISCUSSION THREAD

**Discussion by abdullah-erturk** [rustdesk/rustdesk#3443] [General]
The Nightly version of the Rustdesk software is unfortunately Trojan by Windows Defender.

I use Rustdesk Server in the workplace, and as a client, I use the Nightly version of Rustdesk software. I created automatic script and updated every night, but unfortunately some computers with Windows 11 operating system cannot be installed or updated because of Rustdesk Windows Defender.

![rustdesk_defender](https://user-images.githubusercontent.com/54866587/222075820-253ee6dd-461c-43d4-a2bf-1d8e44af0b11.jpg)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/3164

**Reply by rustdesk** [reply]
Yes, we do not know the exact reason either, so embarrassing.

**Reply by abdullah-erturk** [reply]
I hope you can fix it soon :)

**Reply by rustdesk** [reply]
But no clue yet, :)

**Reply by abdullah-erturk** [reply]
Is it possible for you to report a false alarm to Microsoft?
